### DISCUSSION TITLE
Deploy doesn't work correctly | Netlify

### DISCUSSION THREAD

**Discussion by Alrodr** [thuliteio/doks#230] [Q&A]
Hello everyone,

First of all, thank you for this wonderful repository. 

When I do "npm run start" in local, everything works fine.
When deploying with Netlify, I have the following problem which I show in a GIF.

![problems](https://user-images.githubusercontent.com/45255827/114246717-9856ce00-9993-11eb-999d-a4fd54554699.gif)

- The first problem is that it hides part of the navigation bar in some Docs tabs.
- The second is that it doesn't render correctly the .md in the statistics tab.

My repo: https://github.com/Alrodr/datadoks
My web: https://data-doks.com/

Thank you very much! :)

**Comment by Alrodr** [comment]
Solved. 

thank you so much

**Reply by h-enk** [reply]
@Alrodr, thanks for the compliments and great to hear you got it working now!

For my curiosity and to help others with a similar issue, what did you do to fix it /what was the cause of your problems (I suspect it got something to do with duplicate pages)?

**Reply by Alrodr** [reply]
Of course @h-enk , I had in mind to answer it in detail.

For the first problem: 

> The first problem is that it hides part of the navigation bar in some Docs tabs.

It seems obvious, but it is not... ^^ When you create a new document, make sure that they do not have the same title.

> The second is that it doesn't render correctly the .md in the statistics tab.

In my case, I wanted to create a new section with the same layaout as docs. The following elements need to be well defined for the new section called "docs_new" (For `html `documents, you can take the layout of the `doc `document and change the keyword "docs" to "docs_new".):

```
layaouts/
├── docs_new/
│   ├── list.html
│   └── single.html
├── partials/
│   ├── main/
│   │      ├── docs_new-navigation.html
│   └── sidebar/
│   │      ├──docs_new-toc.html
│   │      ├──docs_new-menu.html
```

If you have any more questions, do not hesitate to write to me and I will be happy to answer you.

Regards

**Reply by h-enk** [reply]
Thanks, that helps /clarifies. Input for updating the documentation/ noted.
